reactioncommerce/reaction
# Zoom image in PDP

## Fix for #3100

This uses the [react-image-magnify](https://github.com/ethanselzer/react-image-magnify) component to render the main image in PDP.

I have tuned some small things like the delay in showing the zoomed image, etc. on a _as I like basis_. Would like to get view of others on this.

### To review
We don't support multiple res. of same image right now. So I am basically zooming the image and showing it to the user. Is this the intended behaviour?

### To test
1. Run `reaction`
2. Upload some image in the `example-product`
3. Hover over the image and see it zoomed.
4. Also try the mobile version which responds to long press instead of hover.
5. Also turn off the edit mode `on` and hover on the image, there should be no zoomed image visible.
5. Log out and log in as a customer, you should be able to see the zoomed image on hover.
